enterprise_access/apps/content_metadata/tests/test_api.py

```diff
@@ -15,10 +16,15 @@ class TestContentMetadataApi(TestCase):
     """
     Tests the content_metadata/api.py functions.
     """
+    def tearDown(self):
+        super().tearDown()
+        # Clear the cache to prevent cache bleed across tests.
+        TieredCache.dangerous_clear_all_tiers()
+
     @mock.patch('enterprise_access.apps.content_metadata.api.EnterpriseCatalogApiClient', autospec=True)
-    def test_get_and_cache_metadata_happy_path(self, mock_client_class):
+    def test_get_and_cache_catalog_content_metadata_happy_path(self, mock_client_class):
         content_keys = ['course+A', 'course+B']
-        customer_uuid = uuid4()
+        catalog_uuid = uuid4()
```

Comment:
A repetitive change in this file is replacing customer_uuid with catalog_uuid.  It was just wrong before.